wgengine/magicsock/magicsock_test.go

```diff
@@ -1530,21 +1530,36 @@ func BenchmarkReceiveFrom(b *testing.B) {
 	for i := range sendBuf {
 		sendBuf[i] = 'x'
 	}
-
 	buf := make([]byte, 2<<10)
-	for i := 0; i < b.N; i++ {
+	return func() {
 		if _, err := sendConn.WriteTo(sendBuf, dstAddr); err != nil {
-			b.Fatalf("WriteTo: %v", err)
+			tb.Fatalf("WriteTo: %v", err)
 		}
 		n, ep, err := conn.ReceiveIPv4(buf)
 		if err != nil {
-			b.Fatal(err)
+			tb.Fatal(err)
 		}
 		_ = n
 		_ = ep
 	}
 }
 
+func TestReceiveFromAllocs(t *testing.T) {
+	t.Skip("cannot enable test until standard package net optimization is available")
+	roundTrip := setUpReceiveFrom(t)
+	avg := testing.AllocsPerRun(100, roundTrip)
```

Comment:
ah, there it is.

maybe remove the t.Skip and lock it in to an expect value as a function of `runtime.Version()`?

Reply:
Done.